js/core/css/respec2.css

```diff
@@ -171,6 +171,64 @@ details.respec-tests-details>li {
   padding-left: 1em;
 }
 
+a[href].self-link:hover {
+  opacity: 1;
+  text-decoration: none;
+  background-color: transparent;
+}
+
+h2,
+h3,
+h4,
+h5,
+h6 {
+  position: relative;
+}
+
+h2>a.self-link,
```

Comment:
Do we really need this level of specificity?

Reply:
I think so, as we might want to add different self-link types.